### PR TITLE
chore(relay): Add a new configuration option: `geoip.path`

### PR DESCRIPTION
**Pull request by olksdr**
Adds new Relay config option.

Related to https://github.com/getsentry/relay/issues/2225 and https://github.com/getsentry/relay/pull/2229

Closes https://github.com/getsentry/relay/issues/2231.

### CODE REVIEW

File: src/docs/product/relay/options.mdx [hunk 1]
```diff
@@ -126,3 +126,9 @@ default.
 | -------------- | ------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | sentry.enabled | Boolean | default: `false` | Determines whether to report internal errors to a separate DSN. `false` means that no internal errors will be sent, but will still be logged.                                        |
 | sentry.dsn     | String  | optional: `true` | Sentry DSN to which you can report internal Relay failures. We recommend setting this to a value that sends Relay errors directly to Sentry instead of sending them to itself first. |
+
+## Geoip
```
Comment (vivianyentran) [inline src/docs/product/relay/options.mdx:130]:
```suggestion
## GeoIP
```

File: src/docs/product/relay/options.mdx [hunk 2]
```diff
@@ -126,3 +126,9 @@ default.
...
+
```
Comment (vivianyentran) [inline src/docs/product/relay/options.mdx:131]: By default, Relay does not include geographical location information in events. Set the MaxMind database file path to add GeoIP info in events.

Comment (vivianyentran) [reply src/docs/product/relay/options.mdx:131]: I think it'd be helpful to add an intro sentence for this section, something like above.

File: src/docs/product/relay/options.mdx [hunk 3]
```diff
@@ -126,3 +126,9 @@ default.
...
+| key        | type   | value           | description                                                                                                              |
+| ---------- | ------ | --------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| geoip.path | String | default: `null` | The path to the MaxMind GeoIP City database file. When set the additional GeoIp information will be added to the events. |
```
Comment (vivianyentran) [inline src/docs/product/relay/options.mdx:134]:
```suggestion
| geoip.path | String | default: `null` | The path to the MaxMind GeoIP City database file. When set the additional GeoIP information will be added to the events. |
```

Does it need to be a MaxMind database? Would other sources work?

Comment (olksdr) [reply src/docs/product/relay/options.mdx]: yes, that's what we currently support.

**Comment by olksdr** [comment]
@vivianyentran addressed all your review comments. PTAL!

**Pull request merged**